gcloud/orb.yaml

```diff
@@ -110,10 +110,10 @@ commands:
               echo >&2 "ERROR: could not find supported package manager"
               exit 1
             fi
-      - run: curl -sSL https://sdk.cloud.google.com | bash -s -- --disable-prompts
-      - run: ln -s /root/google-cloud-sdk/bin/gcloud /usr/bin/gcloud
-      - run: ln -s /root/google-cloud-sdk/bin/gsutil /usr/bin/gsutil
-      - run: ln -s /root/google-cloud-sdk/bin/docker-credential-gcloud /usr/bin/docker-credential-gcloud
+      - run: \[ ! -f /root/google-cloud-sdk \] && curl -sSL https://sdk.cloud.google.com | bash -s -- --disable-prompts
```

Comment:
nit: using multi-line statements might be cleaner here, as it would prevent you from needing to escape these characters:
```yaml
- run: |
    [ ! -f /root/google-cloud-sdk ] && \
    curl -sSL https://sdk.cloud.google.com | bash -s -- --disable-prompts
```

Reply:
That's much cleaner. I've updated the code to something different. But great recommendation.